### DISCUSSION TITLE
How to use `vue-pdf-embed` with `nuxt`?

### DISCUSSION THREAD

**Discussion by hrynko** [hrynko/vue-pdf-embed#114] [Q&A] [answered]
Hi @abdmdjr @Beisenbayev @marcusaram,

Thank you very much for your answers in Nuxt related issues (#91, #21 and #12)!

To summarize and make the solution more discoverable, can I ask one of you to prepare a detailed answer and perhaps some demo snippet? I would really appreciate it.

**Comment by zguig52** [comment, marked as answer]
Quick and easy steps for Nuxt3 integration as a plugin:

1. Create a folder plugins

2. Create a file named `VuePdfEmbed.client.js` inside the folder with following content:

```js
import VuePdfEmbed from 'vue-pdf-embed'

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VuePdfEmbed)
  nuxtApp.vueApp.component('VuePdfEmbed', VuePdfEmbed);
})
```

3. Use it in your app like this:

```vue
<ClientOnly>
  <VuePdfEmbed :source="pdfURL" />
</ClientOnly>
```

**Reply by flamerged** [reply]
Do you know how this is done in Nuxt bridge since it uses vue 2.7
